app/books/models.py

```diff
@@ -102,25 +118,90 @@ def set_book_status(self, status):
         self.status = status
         db.session.commit()
 
-    def __repr__(self):
+    # def __repr__(self):
```

Comment:
It's not a good practice to have commented out code in your project. Removing it will be a good clean up.